modules/apps/remote-app/remote-app-support-web/test/index.js

```diff
@@ -163,4 +163,56 @@ describe('remote-app-support-web', () => {
 			expect(receiveMessage).not.toHaveBeenCalled();
 		});
 	});
+
+	describe('"fetch" command', () => {
+		it('doesn\'t throw an error if "body" is undefined', async () => {
+			const headers = new Headers();
+			headers.append('Content-Type', 'application/json');
+
+			global.fetch = jest.fn(() =>
```

Comment:
I think `fetch.mockResponse` might be the "blessed" way to do this (ie. do a `git grep fetch.mockResponse` and you'll see more usages than direct assignment to `fetch` like this).

Reply:
:+1: 
I knew we had something like that.
Will search and update